tests/fuzzing/roundtrip_fuzzer.py

```diff
@@ -0,0 +1,27 @@
+import sys
+
+import atheris
+from fuzz_helpers import EnhancedFuzzedDataProvider
+
+with atheris.instrument_imports(include=["jupytext"]):
+    import jupytext
+    from jupytext.compare import compare
+
+
+def TestOneInput(data: bytes):
+    fdp = EnhancedFuzzedDataProvider(data)
+
+    py = fdp.ConsumeRemainingString()
+    nb = jupytext.reads(py, "py")
+    py2 = jupytext.writes(nb, "py")
+
+    compare(py, py2)
```

Comment:
Is the objective of this test to make sure that all `.py` files are preserved over round trips? Or do we expect to identify unexpected errors or something like this?